src/GUI/UI/UIGame.cpp

```diff
@@ -325,21 +325,21 @@ GUIWindow_GameVideoOptions::GUIWindow_GameVideoOptions()
     : GUIWindow(WINDOW_VideoOptions, {0, 0}, render->GetRenderDimensions(), 0) {
     // -------------------------------------
     // GameMenuUI_OptionsVideo_Load --- part
-    game_ui_menu_options_video_background = assets->GetImage_ColorKey("optvid", render->teal_mask_16);
-    game_ui_menu_options_video_bloodsplats = assets->GetImage_ColorKey("opvdH-bs", render->teal_mask_16);
-    game_ui_menu_options_video_coloredlights = assets->GetImage_ColorKey("opvdH-cl", render->teal_mask_16);
-    game_ui_menu_options_video_tinting = assets->GetImage_ColorKey("opvdH-tn", render->teal_mask_16);
-
-    game_ui_menu_options_video_gamma_positions[0] = assets->GetImage_ColorKey("convol10", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[1] = assets->GetImage_ColorKey("convol20", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[2] = assets->GetImage_ColorKey("convol30", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[3] = assets->GetImage_ColorKey("convol40", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[4] = assets->GetImage_ColorKey("convol50", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[5] = assets->GetImage_ColorKey("convol60", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[6] = assets->GetImage_ColorKey("convol70", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[7] = assets->GetImage_ColorKey("convol80", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[8] = assets->GetImage_ColorKey("convol90", render->teal_mask_16);
-    game_ui_menu_options_video_gamma_positions[9] = assets->GetImage_ColorKey("convol00", render->teal_mask_16);
+    game_ui_menu_options_video_background = assets->GetImage_ColorKey("optvid", colorTable.TealMask.C16());
+    game_ui_menu_options_video_bloodsplats = assets->GetImage_ColorKey("opvdH-bs", colorTable.TealMask.C16());
+    game_ui_menu_options_video_coloredlights = assets->GetImage_ColorKey("opvdH-cl", colorTable.TealMask.C16());
+    game_ui_menu_options_video_tinting = assets->GetImage_ColorKey("opvdH-tn", colorTable.TealMask.C16());
+
+    game_ui_menu_options_video_gamma_positions[0] = assets->GetImage_ColorKey("convol10", colorTable.TealMask.C16());
```

Comment:
as GetImage_ColorKey implied having a color key, the function itself might have colorkey argument defaulting to teal - will save a lot of text
i think only sprites have different colorkey